sonar_bvt_sdkTypes.hpp

```diff
@@ -12,21 +12,26 @@
 
 namespace sonar_bvt_sdk 
 {
-    struct BvtMonitor
+    namespace image_resolution
```

Comment:
I did it in order to improve the code readability. When I call only sonar_bvt_sdk::High it doesn't say clearly what it is about. With the additional namespace I have to call it as sonar_bvt_sdk::image_resolution::High and then I say that the "High" reffers to the image_resolution.